### ISSUE TITLE
Unable to connect to https://localhost:10443/ 

### ISSUE THREAD

**Issue by FrancisPoe** [EddieDSuza/techwitheddie#4]
I followed your instructions to add Scrypted on the same RasPi as Homebridge. Everything seemed to have installed properly and I 'm even able to access Portainer. However, I am unable to access Scripted at https://localhost:10443/. Everything appears to be running correctly in Portainer. Any help is appreciated.

**Comment by EddieDSuza** [comment]
Can you please check the Docker Container if it's running or stopped. Alternately - you can also try https://localhost:11080/

Additionally restart the server as well

**Comment by FrancisPoe** [comment]
Thanks for the prompt response, I appreciate the assistance! I have taken steps to restart the server and have also ensured that all the Containers are running. Furthermore, I attempted to access Scrypted via https://localhost:11080/ and https://localhost:10443/. Unfortunately, I am still receive an error stating "Unable to connect" in the browser when attempting to access Scrypted. Your guidance on how to resolve this issue is invaluable.

**Comment by EddieDSuza** [comment]
Francis, 

Please use the device IP address to access the scrypted server instead of localhost. so it would be https://192.168.86.34:10443

Alternatively, you need to ensure you have to also reserved the IP address running the scrypted service

**Comment by FrancisPoe** [comment]
I am pleased to inform you that I have resolved the issue! Upon further investigation, I realized that I have a Mesh router setup with two satellites. It appears that Scrypted does not function when connected to a mesh satellite. To address this, I connected my Raspberry Pi directly to the Gateway mesh router, and that did the trick. Thank you very much for your assistance and support in this matter!
